personalitytest/static/js/src/personalitytest-view.js

```diff
@@ -238,6 +238,7 @@ function PersonalityTestXBlockStudent(runtime, element) {
         var handlerUrl = runtime.handlerUrl(element, 'reset_answers');
         $.post(handlerUrl, '{}')
             .done(function () {
+                $('select', element).find('option:selected').removeAttr('selected');
```

Comment:
No this is bad.

The answer I linked to says 

> To deselect all options in a documented, cross-browser manner use either
> 
> ```js
> $("select").val([]);
> ```
> or
> 
> ```js
> // Note the use of .prop instead of .attr
> $("select option").prop("selected", false);
> ```